main.js

```diff
@@ -1,13 +1,40 @@
 var $car = document.querySelector('.car-image');
+$car.style.top = '0';
+$car.style.left = '0';
+var x = 0;
+var y = 0;
+
+var carData = [
```

Comment:
You don't need this in an array, it can just be the object stored within the array